FlickTrainer/app/src/main/java/jp/co/azz/flicktrainer/databases/HistoryDao.java

```diff
@@ -0,0 +1,200 @@
+package jp.co.azz.flicktrainer.databases;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class HistoryDao {
+    private DatabaseHelper dataBaseHelper;
+
+    public HistoryDao(Context context) {
+        dataBaseHelper = new DatabaseHelper(context);
+    }
+
+    //select文のOrderby句に指定する、TOP5件分を取得するための定数
+    private final String TOP_LIMIT5 = " DESC LIMIT 5";
+
+    /**
+     * History テーブルよりスコアTop5 を返却するSELECT文を発行する。
+     *
+     * @return HistoryDto型でSelectされた値を返す。
+     */
+    public List<HistoryDto> selectHistoryScoreTop5() {
+        return this.selectHistory(DatabaseContract.History.COLUMN_NAME_SCORE + TOP_LIMIT5);
+    }
+
+    /**
+     * History テーブルより、プレイ日付Top5 を返却するSELECT文を発行する。
+     *
+     * @return HistoryDto型でSelectされた値を返す。
+     */
+    public List<HistoryDto> selectHistoryPlayDateTop5() {
+        return this.selectHistory(DatabaseContract.History.COLUMN_NAME_PLAY_DATE + TOP_LIMIT5);
+    }
+
+    /**
+     * Historyテーブルをselect文を発行するメソッド（orcerby句を指定する）
+     *
+     * @param orderBy {String} orderby句
+     * @return HistoryDto型でSelectされた値を返す
+     */
+    private List<HistoryDto> selectHistory(String orderBy) {
```

Comment:
条件を付けない場合、全てを取得することがわかるメソッド名をつけてください。
また、ソート順はStringではなく、enumを使うようにしてください。
例えば
```suggestion
    private List<HistoryDto> getAll(Order order) {
```

Reply:
enum型を 使用する形に変更しました。
  メソッド名をgetAll型に変更しました。